### DISCUSSION TITLE
Tail gets disabled automatically. Is there a way to configure this behaviour?

### DISCUSSION THREAD

**Discussion by ricardoboss** [datalust/seq-tickets#1591] [Help] [answered]
Hi there 🙋‍♂️

Seq is a great tool and I would like to introduce it to our company.
One problem we encountered while evaluating Seq was that the "tail" button automatically disables when inspecting an event.

My question is: is there a way to configure this? Can we make "tail" stay on, even when inspecting an event? Thanks and best Regards!

**Comment by nblumhardt** [comment, marked as answer]
Hi Ricardo, thanks for the note!

Currently, no, there's no way to keep tail running when an event is being inspected. I think it might now make more sense for your preference to be the default, though, so I'll keep this in mind and look for an opportunity to try it out.

**Reply by nblumhardt** [reply]
Hi Ricardo,

We looked into this and decided not to make a change right now. When results are coming in quickly, after clicking an event, unless "tail" is stopped, the event can be quickly pushed offscreen. Manually turning tail back on seems like a better option in general, although we can see how in some cases it may be inconvenient.

We'll still keep this in mind and see if there's some general improvement we can make in this area in the future (such as showing a collapsed "N new results" row at the top of the event screen in some circumstances).

Thanks again for the feedback,
Nick

**Reply by ricardoboss** [reply]
Hi Nick,

I imaged a button in the settings, so the users can decide which behavior they want...

Anyhow, the same issue applied when changing the active signals. The events list gets reloaded and the tail button gets disabled. Changing the signals shouldn't stop the tail from running since the attention is on another part of the view anyway. What do you gain from having tail turned off after changing the signals?

Best Regards,
Ricardo

**Reply by nblumhardt** [reply]
Thanks for the follow-up, Ricardo. I see your point regarding signal changing - we'll examine the possibility of making this change, too.

Thanks again,
Nick

**Reply by ricardoboss** [reply]
Hey @nblumhardt, I hope you have been fine. Have you evaluated if you want to change the behavior?

**Reply by nblumhardt** [reply]
Hi @ricardoboss :wave: We're currently putting the finishing touches on a new release, so we haven't been able to loop back around and triage new ideas yet, but it's still on the radar :-)

**Reply by ricardoboss** [reply]
A suggestion from my colleague: look at the Console app on macOS. When you select an event, the bottom half shows details about the event while the stream of events continues to get updates in the top half.

This would also be an elegant solution for Seq, I think.

**Reply by chucker** [reply]
(Console, not Activity Indicator.)
